smalltalksrc/VMMakerTests/VMObjectLayoutTests.class.st

```diff
@@ -61,6 +61,18 @@ VMObjectLayoutTests >> testHeaderOfObjectEncodesTheCorrectAmountOfSlots [
 			equals: objSize ]
 ]
 
+{ #category : #'tests - integerValues' }
+VMObjectLayoutTests >> testNegativeIntegerValuesInRange [
+
+	self assert: (memory isIntegerValue: -17)
+]
+
+{ #category : #'tests - integerValues' }
+VMObjectLayoutTests >> testNegativeIntegerValuesNotInRange [
+
```

Comment:
would be nice to add a little comment such as "an integer smaller than the smallest negative integer is not a valid number"